src/pygeosx/pygeosx.cpp

```diff
@@ -5,7 +5,7 @@
  * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
  * Copyright (c) 2018-2024 Total, S.A
  * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
- * Copyright (c) 2018-2024 Chevron 
```

Comment:
did you run uncrustify on your mac? I think that every time I run it on ruby it changes that whitespace.

Reply:
on ruby...but only these two files....which is strange